gfx/include/public/gfx/gfx.h

```diff
@@ -2,17 +2,74 @@
 
 #pragma once
 
+#include <gfx/gfx_enums.h>
```

Comment:
if ur including these, you don't need these headers to be public. you can just do include "gfx_*.h"

Reply:
This is more for code organization, these are all public headers and the functionality sorted within them are organized into the various headers.